src/components/component-content/component-content.css

```diff
@@ -47,9 +47,14 @@ h3[id^="toc"] + p {
     display: grid;
     grid-template-columns: 1fr 12rem;
     grid-gap: var(--goa-space-2xl);
+
+    @media screen and (max-width: 1230px) {
+        grid-template-columns: 1fr;
+      }
 }
 
 .component-content--content {
     flex: 1 1 auto;
     container: inline-size;
+    min-width: 0;
```

Comment:
Is this needed?

Reply:
Yes, it prevents wide content in `.component-content--content` from [blowing out of the grid column](https://css-tricks.com/preventing-a-grid-blowout/). Mind you, that CSS Tricks article is old so there may be a better method that I'm not aware of.

Reply:
I figured this was some sort of css hack :)